modules/apps/redirect/redirect-test/src/testFunctional/tests/Redirect.testcase

```diff
@@ -942,6 +942,35 @@ definition {
 			locator1 = "Home#PAGE_ACTIVE");
 	}
 
+	@description = "This ensures that the old friendlyURL can be removed from the history."
+	@priority = "5"
+	test RemoveOldFriendlyURL {
+		property osgi.module.configuration.file.names = "com.liferay.layout.admin.web.internal.configuration.FFViewFriendlyURLHistoryConfiguration.config";
```

Comment:
@Jeremy-Ch once our devs enable the feature flag, we won't need this property anymore. Can you create a test improvement ticket to track that for later?